vhdbuilder/packer/components.json

```diff
@@ -40,16 +40,18 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cns:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.4.44.5",
         "v1.4.50",
-        "v1.5.11"
+        "v1.4.51",
```

Comment:
we're dropping 1.4.50 because 51 is already mostly shoved out i assume?

But keeping 1.5.11 because .14 is still baking?

Reply:
dropping 1.4.44.5 because 1.4.50 should be mostly out by the time this goes out, and 1.4.51 is vNext.
keeping 1.5.11 as it's the current release on 1.28+/dualstack, we'll bring 1.5.14 as vNext after freeze

Reply:
Can we only have the latest versions of 1.4.x and 1.5.x?